class/defaults.yml

```diff
@@ -31,3 +31,7 @@ parameters:
           cpu: 1000m
         requests:
           cpu: 500m
+      securityContextCrossplane:
```

Comment:
How can we deal with this in the context of OpenShift?

Reply:
We'd have to remove it. But the upstream default values set it to a value anyway.